Prisma Access logout bug (#26700)
demisto/content: Packs/PrismaAccess/ReleaseNotes/2_0_4.md

#### Integrations

##### Prisma Access

> - Fixed an issue where ***prisma-access-logout-user*** command was failed due to invalid xml.

Reviewer: ```suggestion
- Fixed an issue where ***prisma-access-logout-user*** command failed to run when tenant is provided.
```
